docs/content/guides/operations/kubernetes/overview/index.md

```diff
@@ -86,7 +86,7 @@ softwareupdate --install-rosetta --agree-to-license
 Next, use the following commands to create a new cluster and install the Radius control plane, along with a new environment:
 
 ```bash
-k3d cluster create -p "8081:80@loadbalancer" --k3s-arg "--disable=traefik@server:0"
+k3d cluster create -p "8081:80@loadbalancer" --k3s-arg "--disable=traefik@server:*" --k3s-arg "--disable=servicelb@server:*"
```

Comment:
Just so I know what's going on ... does this command *actually* disable the built-in ingress + load balancer?

Reply:
/cc @Reshrahim @willtsai

Reply:
@rynowak K3d has an external load balancer that can be disabled, but the configuration in this PR does not do that. The external LB is an nginx instance that runs as a container and routes traffic from the host machine into the k3d cluster container. We could disable it by passing `--no-lb` to the `k3d cluster create` command, i.e. `k3d cluster create --no-lib`, however, we need the routing from the host machine into the cluster that is provided by the nginx container.

Inside the k3d cluster there is an internal load balancer called `servicelb` that distributes traffic. That is the one we are disabling with this configuration.